packages/core/src/sequelize-typescript.ts

```diff
@@ -147,6 +147,8 @@ const instanceSequelizeHooks = new HookHandlerBuilder<SequelizeHooks>([
 type TransactionCallback<T> = (t: Transaction) => PromiseLike<T> | T;
 type SessionCallback<T> = (connection: Connection) => PromiseLike<T> | T;
 
+export const SUPPORTED_DIALECTS = ['mysql', 'postgres', 'sqlite', 'mariadb', 'mssql', 'mariadb', 'mssql', 'db2', 'snowflake', 'ibmi'] as const;
```

Comment:
I've started making sure our constants are immutable

```suggestion
export const SUPPORTED_DIALECTS = Object.freeze(['mysql', 'postgres', 'sqlite', 'mariadb', 'mssql', 'mariadb', 'mssql', 'db2', 'snowflake', 'ibmi'] as const);
```